doc/manuals/orion-api.md

```diff
@@ -1514,6 +1514,22 @@ the subscription failed in its last notification or not (i.e. checking that `fai
 The current Orion implementation doesn't support `keyValues` option in `POST /v2/op/notify` operation described in the 
 original NGSIv2 specification. If you attempt to use it you would get a 400 Bad Request error.
 
+## Registration implementation differences
```

Comment:
This new section was created for describing all the differences between Orion and the spec for registrations.

Reply:
Ok. NTC